PIM/Operation/01_ManageProducts.md

```diff
@@ -0,0 +1,304 @@
+  The new product is saved. The *Create product* view is closed. The product list with the new product is displayed.
+
+
+
+## Edit products
+
+After you have created a product, you can edit it. You can either edit a single product or use the mass editing function to edit the same attribute values in several products using a wizard. Depending on the attribute settings, not all attribute values are editable.
+
+### Edit a single product
+
+Edit one or several attribute values for a single product.
+
+#### Prerequisites
+
+At least one product is created, see [Create a product](#create-a-product).
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Product list](../../Assets/Screenshots/PIM/Products/List/Products.png "[Product list]")
+
+1. Click the product you want to edit in the list of products.   
+  The *Edit Product* view is displayed. The *Attributes* tab is preselected.
+
+  ![Attributes](../../Assets/Screenshots/PIM/Products/List/Attributes/AttributesEdit.png "[Attributes]")
+
+2. Edit the desired data of the product in the corresponding fields and tabs.
+
+3. Click the [SAVE] button in the upper right corner.   
+  The changes are saved. The *Edit Product* view is closed.  
+
+
+
+### Mass editing
+
+The mass editing function allows to edit different attribute values of multiple products at the same time.
+
+#### Prerequisites
+
+Several products are created, see [Create a product](#create-a-product).
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Product list](../../Assets/Screenshots/PIM/Products/List/Products.png "[Product list]")
+
+1. Select the checkboxes of the products you want to edit.   
+  The editing toolbar is displayed above the products list.
+
+2. Click the [START MASS EDITING] button in the toolbar.   
+  The *Mass Editing* side bar is displayed on the right side.
+
+  ![Mass editing](../../Assets/Screenshots/PIM/Products/List/MassEditing.png "[Mass editing]")
+
+  > [Info] You can still add products to the mass editing process when the side bar is displayed. Just select the checkboxes of the products in the list and click the [MASS EDITING >] button in the editing toolbar to add the selected products to the mass editing list.    
+  To remove a product from the mass editing list, click the ![Remove](../../Assets/Icons/Trash01.png "[Remove]") (Remove) button right to the respective product. The ![Remove](../../Assets/Icons/Trash01.png "[Remove]") (Remove) button is only displayed if you hover over a product in the list.
+
+3. Click the [START] button in the upper right corner of the side bar to start the mass editing wizard.   
+  The *Select Attributes* wizard window is displayed.
+
+  ![Select attributes](../../Assets/Screenshots/PIM/Products/List/MassEditingWizard/SelectAttributes.png "[Select attributes]")
+
+4. Select the checkboxes of the attributes you want to edit and click the [CONTINUE] button in the bottom right corner.   
+  The *Select Channels and Languages* wizard window is displayed.
```

Comment:
The Select Scopes and Languages window is only displayed, if at least one of the selected attributes is multilanguage/scopeable